action_server/tasks.py

```diff
@@ -101,6 +117,18 @@ def build_frontend(ctx: Context, debug: bool = False, install: bool = True):
         )
 
 
+@task
+def build_binary(ctx: Context) -> None:
+    with _change_to_build_binary():
+        run(ctx, "pyoxidizer", "run", "--release")
+
+
+@task
+def build_go_wrapper(ctx: Context) -> None:
+    with _change_to_build_go_wrapper():
+        run(ctx, "go", "build", "-o", "action-server-unsigned")
```

Comment:
just out of curiosity, was this tested, if it works? I'm asking because it's quite tricky to get this right as files have to be in the right place etc. so probably the wrapper won't actually contain the AS build. Or if it works... I would be really curious how 😄

Reply:
Yes. I've tested this. The `assets` folder seems to be included correctly in the binary.
Give me an example of what I can test locally to make sure this works.

Reply:
oh, did not exact that, awesome then!
And not needed, if building and running works, that's perfect